pkg/runctl/run_manager_test.go

```diff
@@ -2053,7 +2159,10 @@ func (*testHelper1) prepareMocksWithSpec(ctrl *gomock.Controller, spec *stewardv
 }
 
 func newEmptyRunsConfig(ctx context.Context) (*cfg.PipelineRunsConfigStruct, error) {
-	return &cfg.PipelineRunsConfigStruct{}, nil
+	return &cfg.PipelineRunsConfigStruct{
+			TimeoutWait: metav1Duration(time.Minute * 10),
```

Comment:
Should be removed again when the default value is no longer applied by `config.LoadPipelineRunsConfig`.